src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/update/PopulateUpdateSubmission.java

```diff
@@ -0,0 +1,96 @@
+package uk.gov.companieshouse.overseasentitiesapi.service.update;
+
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.DueDiligenceDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.DueDiligence;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.FilingForDate;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.Presenter;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.UpdateSubmission;
+
+/**
+ * This is a helper class for FilingsService when the filing is an update. The purpose of this class
+ * is to populate change submission values into UpdateSubmission.java with the ultimate gaol to
+ * produce a json output of change submissions. It uses OESubmissionDTO and UpdateDTO values to
+ * populate JSON template.
+ */
+public class PopulateUpdateSubmission {
+  private OverseasEntitySubmissionDto overseasEntitySubmissionDto;
+  private UpdateSubmission updateSubmission;
+
+  public PopulateUpdateSubmission(
+      OverseasEntitySubmissionDto overseasEntitySubmissionDto, UpdateSubmission updateSubmission) {
+    this.overseasEntitySubmissionDto = overseasEntitySubmissionDto;
+    this.updateSubmission = updateSubmission;
+  }
+
+  /* method populates values into UpdateSubmission for JSON output */
```

Comment:
Could you make this a Javadoc comment 
```suggestion
  /**
   * This method populates values into UpdateSubmission for JSON output.
   * @return A populated UpdateSubmission object.
   */
```